modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -94,14 +90,13 @@ data:
       remoteWrite:
       - url: "https://remote-write-endpoint.example.com"
         <endpoint_authentication_credentials>
-        <your_write_relabel_configs> <1>
+        <your_write_relabel_configs> #<1>
 ----
 <1> The write relabel configuration settings.
 +
-For `<your_write_relabel_configs>` substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint.
-+
-The following sample shows how to forward a single metric called `my_metric`:
+For `<your_write_relabel_configs>`, substitute a list of write relabel configurations for metrics that you want to send to the remote endpoint:
```

Comment:
To get around leaving `<your_write_relabel_configs>` alone in running text, could you refer to `writeRelabelConfigs` instead?